sound/soc/sof/core.c

```diff
@@ -23,8 +23,8 @@ MODULE_PARM_DESC(sof_debug, "SOF core debug options (0x0 all off)");
 #define SOF_CORE_ENABLE_TRACE BIT(0)
 
 /* SOF defaults if not provided by the platform in ms */
-#define TIMEOUT_DEFAULT_IPC_MS  5
-#define TIMEOUT_DEFAULT_BOOT_MS 100
+#define TIMEOUT_DEFAULT_IPC_MS  3000
+#define TIMEOUT_DEFAULT_BOOT_MS 5000
```

Comment:
Yes, but the message really is "please define platform-specific constraints otherwise you will use defaults"

Reply:
@plbossart Ack, point taken, fixed in the new patch,